OneSignalSDK/onesignal/location/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest>
 
     <!-- At a minimum the location module requires course permission, the app has the option
          to also include ACCESS_FINE_LOCATION and/or ACCESS_BACKGROUND_LOCATION -->
```

Comment:
Let's update this comment? 
We can mention that for this module to work, the **app** has to provide at a minimum ACCESS_COARSE_LOCATION, with option to also include ACCESS_FINE_LOCATION and/or ACCESS_BACKGROUND_LOCATION